### PR TITLE
Feature/221 result manager

### LINKED ISSUE #221: Add display manager for group of results corresponding to single mesh
Single mesh in detector setting could contain multiple graphs.
First idea - for every mesh create a tab in "results" view

### PR DESCRIPTION
**Pull request by ostatni5**
close #221 

obraz

### CODE REVIEW

**Comment by grzanka** [comment]
Looks nice. I like the idea of the left menu with detector meshes. In most cases, we will have only a few, so this is a good choice.

As for the list of plots "per detector mesh" I'd prefer a single column. This is motivated by fact that all plots within the mesh share a horizontal axis (i.e. if in the first plot we have 0...40 range, then on the second one the range is the same). I'd keep a second column for some meta-data related to the plot, like name of quantity, list of filters being used, some basic statistics  (like mean and standard deviation or similar) etc.

File: src/JsRoot/JsRootGraph.tsx [hunk 1]
```diff
@@ -1,38 +1,50 @@
...
     useEffect(() => {
         // create example graph
-        const nbinsx = 20;
-        const y = Array.from({ length: nbinsx }, () => Math.floor(Math.random() * 40));
+        const nbinsx = props.data?.value ?? 20;
+        const y = Array.from({ length: nbinsx }, () => Math.floor(Math.random() * 20));
         const x = Array.from({ length: nbinsx }, (v, k) => k);
 
         const h1 = JSROOT.createTGraph(nbinsx, x, y);
+        h1.fTitle = props.data?.title ?? "Histogram title";
```
Comment (grzanka) [inline src/JsRoot/JsRootGraph.tsx:25]:
```suggestion
        h1.fTitle = props.data?.title ?? "Plot title";
```

File: src/WrapperApp/components/ResultsPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    data?: Object
+}
+
+const simulationData = (() => {
+    const data = [];
+    for (let index = 0; index < 5; index++) {
+        const numberOfElements = Math.floor(Math.random() * 50) + 100;
+        const obj = {
+            uuid: index,
+            name: `DetectorMesh${index}`,
```
Comment (grzanka) [inline src/WrapperApp/components/ResultsPanel.tsx:18]:
```suggestion
            name: `Scoring Mesh ${index}`,
```

File: src/JsRoot/JsRootGraph.tsx [hunk 3]
```diff
@@ -1,38 +1,50 @@
...
-    const containerEl = useRef<HTMLDivElement>(null);
     const [obj, setObj] = useState(undefined);
+    const [drawn, setDrawn] = useState(false);
 
 
     useEffect(() => {
         // create example graph
-        const nbinsx = 20;
-        const y = Array.from({ length: nbinsx }, () => Math.floor(Math.random() * 40));
+        const nbinsx = props.data?.value ?? 20;
```
Comment (grzanka) [inline src/JsRoot/JsRootGraph.tsx:20]: rename `nbinsx` to `npoints` as we are not dealing with histograms here

File: src/JsRoot/JsRootGraph.tsx [hunk 4]
```diff
@@ -1,38 +1,50 @@
...
+        const y = Array.from({ length: nbinsx }, () => Math.floor(Math.random() * 20));
         const x = Array.from({ length: nbinsx }, (v, k) => k);
 
         const h1 = JSROOT.createTGraph(nbinsx, x, y);
```
Comment (grzanka) [inline src/JsRoot/JsRootGraph.tsx:24]: Rename `h1` to `gr` to follow ROOT conventions:
https://root.cern.ch/doc/master/classTGraph.html

`h1` suggests we have histogram here

**Comment by ostatni5** [comment]
obraz

**Review by grzanka — APPROVED** [review]
Looks good

**Pull request merged**